docs/api/telegram.rst

```diff
@@ -2164,6 +2253,41 @@ about its business.
       The float latitude as defined by the sender.
 
 
+.. py:class:: botogram.Permissions
+
+   This class represents the permissions of the user.
+   If you use this as a context manager, the save method will automatically be called if no exceptions were raised."
+   [example with only the context manager, not the full handler]
+
+   .. py:attribute:: until_date
+
+      The unix timestamp or datime format of when the changes you're doing will be reverted.
+
+   .. py:attribute:: send_messages
+
+      The boolean value if the user can send messages.
+
+   .. py:attribute:: send_media_messages
+
+      The boolean value if the user can send media messages.
+
+   .. py:attribute:: send_other_messages
+
+      The boolean value if the user can send other messages.
+
+   .. py:attribute:: add_web_page_previews
+
+      The boolean value if the user can send web page previews.
+
+   .. py:method:: save()
+
+      Send the changes to Telegram.
+
+      This method automatically detects the changes you made and doesn't do anything if no attribute was changed.
+
+      .. versionadded:: 0.6
```

Comment:
This versionadded should be at the top of the class definition, since the whole class was added this release.